e2e/e2e-local.sh

```diff
@@ -13,4 +13,4 @@ export LOCATION
 export AZURE_TENANT_ID
 
 go version
-go test -timeout $TIMEOUT -v -run Test_All ./
\ No newline at end of file
+go test -parallel 100 -timeout $TIMEOUT -v -run Test_All ./
```

Comment:
can we also allow for the parallel count to be set as an ENV with default 100 is not set.